### PR TITLE
Fix: Update shorebird.bat - CLI wasnt working, fixed

### PR DESCRIPTION
**Pull request by Musaddiq625**
Just installed the shorebird CLI, having these errors:

on entering 'shorebird' having this error "& was unexpected at this time" also having "Could not find a command named "234"."

## Type of Change

- [ ] ✨ New feature (non-breaking change which adds functionality)
- [x] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

File: bin/shorebird.bat [hunk 1]
```diff
@@ -18,4 +18,4 @@ SET PowerShellScriptPath=%CurrentDirectory%shorebird.ps1
 
 REM Pass up to four arguments to the PowerShell script
 REM This number is arbitrary and can be increased if needed
-& %powershell_executable% -NoProfile -ExecutionPolicy Bypass -Command "& '%PowerShellScriptPath%' %1% %2% %3% %4%";
```
Comment (eseidel) [inline bin/shorebird.bat:21]: Should we be concerned that `shorebird` on windows only passes along the first 4 arguments? :) @bryanoltman

Comment (bryanoltman) [reply bin/shorebird.bat:21]: Yeah, there's probably a better way to do this :)

Comment (bryanoltman) [reply bin/shorebird.bat:21]: I believe I was referencing https://superuser.com/a/1630441

**Comment by bryanoltman** [comment]
@Musaddiq625 Can you provide the command you ran and the full output from your console to help us debug this?

**Comment by Musaddiq625** [comment]
@bryanoltman 
so sorry Bryan, missed this notification (gmail issue)
I followed the documentation:
https://docs.shorebird.dev/#:~:text=%24-,shorebird,-The%20shorebird%20command

I was just trying to detect the 'shorebird' keyword in the terminal, I even restarted the system but it didnt solve the issue
I closed this issue because when I did some research I saw that %n% was to detect multiple arguments
then I tried after removing one or two %n% but some more commands need those arguments

and then I thought that, that could be my mistake I missed some configuration
and **because its totally paid** ._. I didnt try after that

**Comment by bryanoltman** [comment]
Sorry to hear that didn't work. Could you tell me more about you system? What version of Windows are you running? Do you have Powershell installed?

**Comment by Musaddiq625** [comment]
yes I do have powershell installed with version 5.1.19041.3031
Windows 10 Pro (Version 22H2)
OS Build: 19045.3086

**Comment by bryanoltman** [comment]
Thanks for the info! Not seeing any obvious problems yet. Where is shorebird installed on your system? What do you see when you run `where.exe shorebird`? Where are you executing this command from (i.e., what is the current working directory when you run a shorebird command)?

**Comment by Musaddiq625** [comment]
for now it isnt installed in my system
I'll try again and will get back with the update

**Pull request closed without merging**